website/src/components/Header/index.tsx

```diff
@@ -32,8 +32,8 @@ const Header: React.FC = () => {
     const displaySocialLinks = () => {
           return (
               <div className={classes.socialIconsWrapper}>
-                    <Link href={EXTERNAL_LINKS.OPENEBS_GITHUB_REPO}>
-                        <img loading="lazy" src="/images/logos/githubLogo.svg" className={classes.socialIcon} alt={t('generic.github')}/>
+                    <Link href={EXTERNAL_LINKS.OPENEBS_GITHUB_REPO} target="_blank" rel="noopener">
+                        <img loading="lazy" src="../images/logos/githubLogo.svg" className={classes.socialIcon} alt={t('generic.github')}></img>
```

Comment:
Why a closing tag for img ?

Reply:
I guess some rebase issues caused this, will check and push again.

Reply:
Sure will remove the closing tag.

Reply:
There was some rebase issue, have resolved them.